eth/api.go

```diff
@@ -1360,7 +1422,7 @@ func (s *PublicTransactionPoolAPI) SignTransaction(args SignTransactionArgs) (*S
 // the accounts this node manages.
 func (s *PublicTransactionPoolAPI) PendingTransactions() []*RPCTransaction {
 	pending := s.txPool.GetTransactions()
-	transactions := make([]*RPCTransaction, 0)
+	var transactions []*RPCTransaction
```

Comment:
If you want to modify this, a better version would be `make([]*RPCTransaction, 0, len(pending))`, as that would prevent GC overhead due to constantly growing the array from a size of zero up to the pending number.

Btw, what is the role of these methods? Are these used currently? The name seems a bit misinforming to me, as it doesn't actually return all the pending transactions (that I would expect from the name), rather only those made by me.